api/src/main/java/com/datatorrent/api/ControlTupleEnabledSink.java

```diff
@@ -18,7 +18,7 @@
  */
 package com.datatorrent.api;
 
-import org.apache.apex.api.UserDefinedControlTuple;
+import org.apache.apex.api.ControlTuple;
```

Comment:
Yes, moving ControlTuple to a package that reflects the functionality of user/operator defined tuple is acceptable.